variables.tf

```diff
@@ -183,16 +183,16 @@ variable "vpc_flow_logs" {
     log_destination_type = string
     retention_in_days    = optional(number)
     tags                 = optional(map(string))
-    traffic_type         = optional(string)
+    traffic_type         = optional(string, "ALL")
     destination_options = optional(object({
-      file_format                = optional(string)
-      hive_compatible_partitions = optional(bool)
-      per_hour_partition         = optional(bool)
+      file_format                = optional(string, "plain-text")
+      hive_compatible_partitions = optional(bool, false)
+      per_hour_partition         = optional(bool, false)
     }))
   })
 
   default = {
-    log_destination_type = "none"
+    log_destination_type = "cloud-watch-logs"
```

Comment:
By default we probably do not want to build flow logs since they can get quite expensive